pepper-apis/src/main/resources/db-changes/schema/DDP-4458-truncate-varchar-for-utf8mb4.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20200330-truncate-varchar-for-utf8mb4">
+        <modifyDataType tableName="umbrella_study" columnName="study_name" newDataType="varchar(200)"/>
```

Comment:
This `varchar` column is used in a unique key index and mysql limits that to something like 3KB. Originally at `varchar(1000)` (!!), 1000 * 4 bytes is 4000 bytes...